rasa/__main__.py

```diff
@@ -64,7 +66,20 @@ def create_argument_parser() -> argparse.ArgumentParser:
 
 
 def print_version() -> None:
-    print("Rasa", version.__version__)
+    """Prints version information of rasa tooling and python."""
+    python_version, os_info = sys.version.split("\n")
+    try:
+        from rasax.community.version import __version__  # pytype: disable=import-error
+
+        rasa_x_info = __version__
+    except ModuleNotFoundError:
+        rasa_x_info = None
+    print(f"Rasa Version     : {version.__version__}")
+    print(f"Rasa SDK Version : {rasa_sdk_version}")
+    print(f"Rasa X Version   : {rasa_x_info}")
+    print(f"Python Version   : {python_version}")
+    print(f"Operating System : {platform.system()}")
```

Comment:
why not `platform.platform()`?

Reply:
It gives slightly more information that could potentially be relevant, like that the system is 64bit. It's not super needed but I figured it was more complete.

Reply:
Wait, `platform.system()` does? Because for me:
```
In [1]: import platform

In [2]: platform.platform()
Out[2]: 'Darwin-19.3.0-x86_64-i386-64bit'

In [3]: platform.system()
Out[3]: 'Darwin'
```
I would also like the slightly more information :)

Reply:
Crud, mybad! Somehow I merged the whole thing with `system` instead of `platform`. I've added a quick fix PR here: https://github.com/RasaHQ/rasa/pull/5793